theme/templates/pages/dashboard.html

```diff
@@ -0,0 +1,321 @@
+{% extends "base.html" %}
+
+{% load mezzanine_tags static blog_tags accounts_tags dashboard_tags i18n %}
+
+{% comment %}
+
+These data should be given fast without ajax.
+The dashboard template assumes context dictionary has the following keys
+timestamp
+title
+first author
+resource id
+content type
+privacy;;
+
+timestamp should go with a template tags to convert to a human readble day.
+{% endcomment %}
+
+{% block main %}
+    <div class="container  collapse dashboard_section_container" id="getStarted">
+        {% trans "Getting Started" as whole_title %}
+        {% include "includes/apps_block_title.html" %}
+        <div id="row-1" class="row">
+            <div class="col-xs-12 col-sm-4  get_started_cell">
+                <a href="https://help.hydroshare.org/introduction-to-hydroshare/getting-started/"
+                   class="get_started_link">Check out tutorials</a>
+            </div>
+            <div class="col-xs-12 col-sm-4  get_started_cell">
+                <a href="/user/" class="get_started_link">Complete profile</a>
+            </div>
+            <div class="col-xs-12 col-sm-4  get_started_cell">
+                <a href="/search/" class="get_started_link">Browse data and models</a>
+            </div>
+        </div>
+
+        <div id="row-2" class="row">
+            <div class="col-xs-12 col-sm-4 get_started_cell">
+                <a href="https://help.hydroshare.org/introduction-to-hydroshare/" class="get_started_link">How does
+                    HydroShare work</a>
+            </div>
+            <div class="col-xs-12 col-sm-4 get_started_cell">
+                <a href="https://help.hydroshare.org/introduction-to-hydroshare/getting-started/"
+                   class="get_started_link">Review an example
+                    resource</a>
+            </div>
+            <div class="col-xs-12 col-sm-4 get_started_cell">
+                <a href="https://help.hydroshare.org/introduction-to-hydroshare/getting-started/"
+                   class="get_started_link">What's
+                    new</a>
+            </div>
+        </div>
+
+    </div>
+
+
+    <div>
+        <div class="container dashboard_section_container">
+            <div class="row dashboard_section_container_title_row">
+                <span>
+                 <h3 class="home-page-block-title">Recently Visited </h3>
```

Comment:
across all html and js files please check your whitespaces (length and type)

Reply:
Will be updated with "-" for all CSS selector in next push.